sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2022-2023.bpmn

```diff
@@ -483,516 +519,423 @@ studentCalculatedTotalIncome</bpmn:text>
         <zeebe:ioMapping>
-          <zeebe:output source="=if partner1StudentStudyWeeks = null then 0 else partner1StudentStudyWeeks" target="calculatedDataPartnerStudentStudyWeeks" />
+          <zeebe:output source="=calculatedDataStudentPartnerInformationAndIncome.calculatedDataRelationshipStatus" target="calculatedDataRelationshipStatus" />
+          <zeebe:output source="=calculatedDataStudentPartnerInformationAndIncome.calculatedDataPartner1TotalIncome" target="calculatedDataPartner1TotalIncome" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_1d1sl5k</bpmn:incoming>
-      <bpmn:outgoing>Flow_02m6u8j</bpmn:outgoing>
-    </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_02st413" name="calculatedDataPartner1TotalIncome">
+      <bpmn:incoming>Flow_0cdutv1</bpmn:incoming>
+      <bpmn:outgoing>Flow_0qhg5p8</bpmn:outgoing>
+    </bpmn:scriptTask>
+    <bpmn:intermediateThrowEvent id="Event_1du7pjb" name="Calculate Eligible Dependents">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=studentDataEstimatedSpouseIncome" target="calculatedDataPartner1TotalIncome" />
+          <zeebe:output source="=count(&#10;    calculatedDataDependants[&#10;      // 0-18 years of age: eligible.&#10;      date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;      // 19-22 years of age: post-sec must be Yes or Tax is Yes.&#10;      or (&#10;        date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;        and date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;        and (item.attendingPostSecondarySchool = &#34;yes&#34; or item.declaredOnTaxes = &#34;yes&#34;)&#10;      )&#10;      // &#62; 22 years: eligible if Tax is Yes.&#10;      or (&#10;        date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;        and item.declaredOnTaxes = &#34;yes&#34;&#10;      )&#10;    ]&#10;  )" target="calculatedDataTotalEligibleDependants" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_1fbmivh</bpmn:incoming>
-      <bpmn:outgoing>Flow_0n7won7</bpmn:outgoing>
+      <bpmn:incoming>Flow_0dqxcli</bpmn:incoming>
+      <bpmn:outgoing>Flow_0exl4fg</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_00c1iyh" name="calculatedDataPartner1TotalIncome">
+    <bpmn:intermediateThrowEvent id="Event_0hxxmu9" name="Define Total Income">
```

Comment:
Would it not be the case of moving the "Define Total Income" logic after the check for single/married, as below?
![image](https://github.com/bcgov/SIMS/assets/61259237/e0d92ca1-ad18-4485-923a-2b7a1f04cee4)

Reply:
I changed this way:
![image](https://github.com/bcgov/SIMS/assets/78114138/c85894b7-6b44-45cc-8aaf-f7766538f600)
This way we don't even need to check CRA reported income when single. Thanks!